src/app/features/post/post.component.ts

```diff
@@ -113,9 +121,29 @@ export class PostComponent {
     private router: Router,
     private markdownService: MarkdownService,
     @Inject(DOCUMENT) private document: Document,
-    private htmlInMarkdownService: HtmlInMarkdownService
+    private htmlInMarkdownService: HtmlInMarkdownService,
+    private postUrlPipe: PostUrlPipe,
+    private meta: Meta
   ) {}
 
+  ngOnInit(): void {
+    this.post$
+      .pipe(
+        filter((value): value is Post => !!value),
+        map(value => this.postUrlPipe.transform(value, value.teaser)),
+        takeUntil(this.destroy$)
+      )
+      .subscribe(content => {
+        this.meta.updateTag({ name: 'og:image', content });
+        this.meta.updateTag({ name: 'og:url', content });
```

Comment:
`og:url` is the current URL, not the image's. Also, both need to be absolute

Reply:
It's also outdated